### DISCUSSION TITLE
volume control

### DISCUSSION THREAD

**Discussion by KeesKemper** [alexkn/node-red-contrib-dlna#7] [Ideas]
It would be nice to have commands to control the volume.
Is someone able to add this functionality?

**Comment by alexkn** [comment]
implemented in [v0.2.0](https://github.com/alexkn/node-red-contrib-dlna/releases/tag/v0.2.0)

**Reply by KeesKemper** [reply]
After having installed the new version in noder red I get the following error:

Flows stopped due to missing node types.

dlna-action

**Reply by alexkn** [reply]
should be fixed with [v0.2.1](https://github.com/alexkn/node-red-contrib-dlna/releases/tag/v0.2.1)

**Reply by KeesKemper** [reply]
With v0,2,1 I get the following error for every DLNA action:
"Error: device url not found"

**Reply by alexkn** [reply]
Looks like your device could not be discovered via SSDP.
But that works on my system with v0.2.1.
Please do not reuse a discussion thread for two different problems, open one Discussion per Idea/Problem.
